### ISSUE TITLE
RuntimeError: <function name at 0x113970398> not installed on your system

### ISSUE THREAD

**Issue by mrgloom** [bootphon/phonemizer#20]
Traceback (most recent call last):
  File "generate.py", line 177, in <module>
    main()
  File "generate.py", line 132, in main
    txt = text2phone(args.text, char2code)
  File "generate.py", line 52, in text2phone
    ph = phonemize(text, separator=seperator)
  File "build/bdist.macosx-10.13-x86_64/egg/phonemizer/phonemize.py", line 90, in phonemize
  File "build/bdist.macosx-10.13-x86_64/egg/phonemizer/backend.py", line 263, in __init__
  File "build/bdist.macosx-10.13-x86_64/egg/phonemizer/backend.py", line 57, in __init__
RuntimeError: <function name at 0x113970398> not installed on your system

**Comment by mrgloom** [comment]
Looks like it's bacause `festival` , `brew install festival` don't work.
I was able to build festival from source, but where to put binaries?

Looks related:
https://github.com/bootphon/phonemizer/issues/5

**Comment by mrgloom** [comment]
I have added `festival` binary to `PATH` and now it works:

```
export PATH=$PATH:/Users/my_user/external_projects/text-to-speech/festival/festival/src/main/

/Users/my_user/external_projects/text-to-speech/festival/festival/src/main/
├── Makefile
├── audsp.cc
├── audsp.o
├── festival
├── festival_client
├── festival_client.cc
├── festival_client.o
├── festival_main.cc
├── festival_main.o
└── make.depend
```

But I wonder what is the difference between `festival`, `espeak`, `segments`, is it possible to switch them using python API?

Looks like they are backends for `phonemize` binary
`phonemize: error: argument -b/--backend: invalid choice: '/Users/my_user/external_projects/text-to-speech/festival/festival/src/main/festival' (choose from 'espeak', 'festival', 'segments')`

**Comment by mmmaat** [comment]
Hi, of course you can choose the backend you want from the API with the `backend` argument, see https://github.com/bootphon/phonemizer/blob/5069681148f811a319bf1d2023bcb8cd0a2211e6/phonemizer/phonemize.py#L30

When you have `python setup.py install` the phonemizer, have a `phonemize --version` to detail the backends you have on your machine:
```
$ phonemize --version
phonemizer-1.0.1
available backends: festival-2.5.0, espeak-ng-1.49.3, segments-2.0.1
```
